src/content/migrate/5.md

```diff
@@ -146,7 +147,7 @@ Deprecation warnings. You might get a lot of deprecation warnings. This is not a
 
 #### Turn off ES2015 syntax in runtime code, if necessary
 
-By default, webpack's runtime code uses ES2015 syntax to build smaller bundles. If your build targets environments that don't support this syntax (like IE11), you'll need to set `output.ecmaVersion: 5` to revert to ES5 syntax.
+By default, webpack's runtime code uses ES2015 syntax to build smaller bundles. If your build targets environments that don't support this syntax (like IE11), you'll need to set `target: ["web", "es5"]` to revert to ES5 syntax (`web` if target environment is browser).
```

Comment:
```suggestion
By default, webpack's runtime code uses ES2015 syntax to build smaller bundles. If your build targets environments that don't support this syntax (like IE11), you'll need to set `target: ['web', 'es5']` to revert to ES5 syntax (`web` if target environment is browser).
```

Reply:
We are using single quotes in JS snippets in webpack.js.org, changing for consistency